### PR TITLE
minor edits for formatting, clarity, consistency, and capitalisation

### PR DESCRIPTION
**Pull request by christinerose**
Signed-off-by: Christine Rose <professor.rose@gmail.com>

Please check to see all the "opam"s are formatted to differentiate between opam, the package manager, and `opam` the command. There were some in there I wasn't sure about. I'll leave comments where.

### CODE REVIEW

File: doc/opam-provided.mld [hunk 1]
```diff
@@ -11,9 +11,9 @@ leaf-dependencies and those that don’t have dependency intersections with
 vendored packages. Due to possibly of unexpected interactions, this feature is
 only meant for advanced users, and the solutions it produces might be in flux.
 
-{2 Usecases}
+{2 Use Cases}
 
-Sometimes it is not possible to put all your dependencies into the [duniverse],
+Sometimes it's not possible to put all your dependencies into the [duniverse],
 be it due to your dependencies not building with [dune] or some tooling that
 should be just installed via [opam]. Such examples include:
```
Comment (christinerose) [inline doc/opam-provided.mld:18]: this `opam` seems clear that you're talking about the command, correct?

File: doc/opam-provided.mld [hunk 2]
```diff
@@ -25,26 +25,26 @@ should be just installed via [opam]. Such examples include:
...
+i.e., requiring all the dependencies of your opam files to be able to be
 built as part of the [duniverse].
 
-As such it is only recommended that packages are installed via [opam] for which
+As such it's only recommended that packages are installed via [opam] for which
 there is no way to build them with [dune], e.g., by using the
 {{:https://github.com/dune-universe/opam-overlays}opam-overlays} repository
 with Dune ports.
 
 To let [opam-monorepo] know a package is to be installed via [opam], it needs
```
Comment (christinerose) [inline doc/opam-provided.mld:39]: "to be installed via `opam`" ... the command? or the package manager?
I suspect the command, so it's formatted correctly if that's right.

File: doc/opam-provided.mld [hunk 3]
```diff
@@ -25,26 +25,26 @@ should be just installed via [opam]. Such examples include:
...
-to be marked as such in the Opam file. For this, it needs to be added as normal
+to be marked as such in the opam file. For this, it needs to be added as normal
 in the [depends] field, as well in addition into the new
 [x-opam-monorepo-opam-provided] field. This field takes a single package or a
 list of packages to be installed via [opam] instead of being pulled into the
```
Comment (christinerose) [inline doc/opam-provided.mld:43]: "installed via `opam`" command or pkg manager as a whole?

File: doc/opam-provided.mld [hunk 4]
```diff
@@ -84,23 +84,23 @@ $ opam install --ignore-pin-depends --deps-only ./ --locked
...
 The [opam]-provided subset of dependencies is then {e excluded} from the
 [duniverse] packages.
 
-{3 Resolution of dependency overlaps}
+{3 Resolution of Dependency Overlaps}
 
 There are cases where a dependency package appears multiple times in the
-project’s dependency tree. It’s possible that a package is part of the packages
-to be included in the [duniverse], as well as packages installed by [opam].
+project’s dependency tree. It’s possible that a package is part of the packages
```
Comment (christinerose) [inline doc/opam-provided.mld:102]: "installed by `opam`" ... command or pkg manager as a whole?

**Review by NathanReb — APPROVED** [review]
The distinction between opam the command and opam the package manager is not always easy but here it clearly is the tool/the command as we want to distinguish packages that the user will install using `opam monorepo pull` and the ones they will install using `opam install`.

**Comment by christinerose** [comment]
I think I accepted your suggestion, but I also changed it in my branch. I think it's ready to merge, so please let me know if I missed a step. :)

**Pull request merged**